tests/test_alert.py

```diff
@@ -0,0 +1,42 @@
+from unittest import TestCase
+from unittest.mock import patch
+from thousandeyessdk import ThousandEyes as TE
+from thousandeyessdk.enum import AlertType
+
+from . import ALERT_WITH_AGENTS, ALERT_WITH_MONITORS, USERNAME, AUTH_TOKEN, URL, USERNAME
+
+@patch('thousandeyessdk.requests.request')
+class TestAlert(TestCase):
+    def test_alert_with_agents(self, response_mock):
```

Comment:
response_mock is an request object mock so name it accordingly, I suggest m__request

Reply:
corrected a6c07d7